### ISSUE TITLE
Unable to build 0.7.2 for armv7

### ISSUE THREAD

**Issue by ggardet** [parallaxsecond/parsec#378]
0.6.0 builds fine for armv7, but I am unable to build 0.7.2 for armv7 due to:
```
[  179s] error: failed to run custom build command for `tss-esapi-sys v0.1.1`
[  179s] 
[  179s] Caused by:
[  179s]   process didn't exit successfully: `/home/abuild/rpmbuild/BUILD/parsec-0.7.2/target/release/build/tss-esapi-sys-2076e3c7eb1fcc79/build-script-build` (exit code: 101)
[  179s]   --- stderr
[  179s]   thread 'main' panicked at 'Compilation target (armv7-unknown-linux-gnueabihf) is not part of the supported targets (["x86_64-unknown-linux-gnu", "aarch64-unknown-linux-gnu", "armv7-unknown-linux-gnueabi", "arm-unknown-linux-gnueabi"]). Please compile with the "generate-bindings" feature or add support for your platform :)', /home/abuild/rpmbuild/BUILD/parsec-0.7.2/vendor/tss-esapi-sys/build.rs:34:13
```

**Comment by hug-dev** [comment]
Thanks for the report!
If you can modify features, you can compile adding the feature `"tss-esapi/generate-bindings"` to go over that bug.

**Comment by hug-dev** [comment]
Your immediate bug will be fixed with #381 (hopefully). It's a bit more tricky to add cross-compilation of the TPM provider as one needs to also cross-compile the TSS libraries. It will be looked at in #382.
The `tss-esapi` crate has its own cross-compilation tests in [here](https://github.com/parallaxsecond/rust-tss-esapi/blob/main/tss-esapi/tests/cross-compile.sh).

**Comment by hug-dev** [comment]
That should be fixed in the latest `main` but feel free to say if not and we will re-open :)
